core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -659,6 +650,33 @@ public String send(MessageProducer messageProducer, javax.jms.Message message, b
 		}
 	}
 
+	protected void logMessageDetails(javax.jms.Message message, MessageProducer messageProducer) throws JMSException {
+		if (log.isInfoEnabled() && !log.isDebugEnabled()) {
+				log.info("[" + getName()
+						+ (messageProducer != null ? " ] message destination: [" + messageProducer.getDestination() + "] " : "")
+						+ "] msgID: [" + message.getJMSMessageID()
+						+ "] correlationID: [" + message.getJMSCorrelationID()
+						+ "] using deliveryMode: [" + message.getJMSDeliveryMode() + "] "
+						+ (message.getJMSReplyTo() != null ? "replyTo: [" + message.getJMSReplyTo() + "]" : ""));
+				return;
+		}
+		if (!log.isDebugEnabled()) {
+			return;
+		}
+
+		log.debug(getLogPrefix() + "sender on [" + getDestinationName()
+				+ "] JMSDeliveryMode=[" + message.getJMSDeliveryMode()
+				+ "] \n  JMSMessageID=[" + message.getJMSMessageID()
+				+ "] \n  JMSCorrelationID=[" + message.getJMSCorrelationID()
+				+ "] \n  JMSTimestamp=[" + DateUtils.format(message.getJMSTimestamp())
+				+ "] \n  JMSExpiration=[" + message.getJMSExpiration()
+				+ (message.getJMSPriority() != 0 ? "] \n  JMSPriority=[" + message.getJMSPriority() : "")
+				+ (message.getJMSType() != null ? "] \n  JMSType=[" + message.getJMSType() : "")
+				+ (message.getJMSReplyTo() != null ? "] \n  JMSReplyTo=[" + message.getJMSReplyTo() : "")
```

Comment:
Wij gebruiken nergens (anders) newlines in de logging dus misschien dat het verstandig is het hier gelijk te trekken (en dus weg te halen).
Ook mogen deze waardes gelogd worden wanneer ze 0 zijn :)

Reply:
Oke, dat wist ik niet. Ik had het gewoon laten staan. Ik ruim de \n nu op. De JMSType en ReplyTo zijn niet altijd gevuld.. die houd ik liever wel conditioneel, om niet te veel zut te loggen.